HLTrigger/Configuration/python/HLT_75e33/tasks/HLTHgcalTiclPFClusteringForEgammaUnseededTask_cfi.py

```diff
@@ -13,14 +12,12 @@
 from ..modules.particleFlowSuperClusterHGCalFromTICLUnseeded_cfi import *
 from ..modules.ticlLayerTileProducer_cfi import *
 from ..modules.ticlSeedingGlobal_cfi import *
-from ..modules.ticlTrackstersEMForEgamma_cfi import *
-from ..modules.ticlTrackstersHADForEgamma_cfi import *
```

Comment:
Can files like `ticlTrackstersEMForEgamma_cfi` be removed then?

Reply:
indeed, they are not being used anywhere.. so can be removed.
I'll clean it up then